schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -1592,6 +1596,212 @@
         "settings",
         "protectedSettings"
       ]
+    },
+    "VMSnapshot":{
```

Comment:
The two definitions don't follow the right pattern. Please look at the other extension definitions to correct it.